examples/svm/plot_custom_kernel.py

```diff
@@ -17,26 +17,26 @@
 iris = datasets.load_iris()
 X = iris.data[:, :2]  # we only take the first two features. We could
                       # avoid this ugly slicing by using a two-dim dataset
-Y = iris.target
+y = iris.target
 
 
-def my_kernel(X, Y):
+def my_kernel(X, y):
```

Comment:
The *y* here is not the same as the *y* on line 20. This one should remain capitalized `Y`, like the kernels in the pairwise module.